packages/thegraph-data-access/.nycrc

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Is this file versioned on purpose?

Reply:
yes, we have this on every package... nyc is the tool for test coverage